.github/SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+As careful and community-oriented maintainers, we strive to keep our code safe to use for everyone. 
+For this purpose, we highly value your input and will acknowledge your contributions as best we can.
+
+## Supported Versions
+
+Unfortunately, our team size does not permit us to support multiple versions at once. 
```

Comment:
“Unfortunately, the resources available for maintaining `message_ix` do not allow us to:

- Support multiple versions at once.
- Publish maintenance updates for older versions. For example, once `message_ix` version 3.10 is released, no further patch versions like 3.9.x, 3.8.x, etc. are possible.”